External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -234,4 +277,28 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 end
 
 
+%% ====================================================================================
+%% 8. Create JSON sidecar
+if bSaveJson
+    % a. First, we load the reference sidecar JSON file, if it exists
+    if bCopyOrigJson && xASL_exist(pathOrigJson, 'file')
+        json = xASL_io_ReadJson(pathOrigJson);
+    elseif bCopyOrigJson
+        warning([pathOrigJson ' did not exist']);

```

Comment:
it will not really crash. but it would be better to also do json=struct() here

Reply:
Fixed in